src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureKeyVaultReference/AzureKeyVaultSecretProvider.cs

```diff
@@ -63,6 +82,38 @@ public async Task<string> GetSecretValue(Uri secretUri, CancellationToken cancel
             return secretValue;
         }
 
+        internal bool AnyExpiredSecrets()
```

Comment:
>internal [](start = 8, length = 8)

It's an internal class. If the methods are meant to be called outside of the class it's easier to understand if they're public.